### PR TITLE
Log upstream req/resp that we can calculate response time

### PR DESCRIPTION
**Pull request by ruhan1**
This is for [MMENG-3652](https://issues.redhat.com/browse/MMENG-3652). Add some log messages.

### CODE REVIEW

File: src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java [hunk 1]
```diff
@@ -341,7 +335,7 @@ public void onFailure( @NotNull Call call, @NotNull IOException e )
                         scope.close();
                         span.end();
 
-                        logger.trace( "Failed: " + call.request().url(), e );
+                        logger.debug( "Failed: [" + method + "] " + url, e );
```
Comment (sswguo) [inline src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java:338]: Do we need the timestamp to match the request and response ? like this example `1700621504091.392790605444873` in content service:
` [GET /folo/track/build-A4S2BBDNVVIAA/maven/group/build-A4S2BBDNVVIAA/org/slf4j/slf4j-api/1.7.28/slf4j-api-1.7.28.jar (1700621504091.392790605444873)]`

that maybe also help to debug some issues. 

And I think it's worth to make it `INFO` level.

Comment (ruhan1) [reply src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java]: The log itself has time stamp, like
`2023-11-21 16:08:55,497 DEBUG [org.com.uti.gat.ser.ProxyService_Subclass] (OkHttp http://indy.indy-stage.svc.cluster.local/...) Setting response header: Expires = 0
`
It is down to milliseconds. Do you think it is good enough? or you mean something else?

INFO is Ok.  Updated.

Comment (sswguo) [reply src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java]: Sorry for the confusing, I mean the timestamp we can identify the related request and response, if there are two same requests ( rare case ) coming, we can use this to check the related response for each.

Comment (ruhan1) [reply src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java]: I see. The thing like '(1700621504091.392790605444873)' is printed by %thread. Not sure why Quarkus truncate the thread name with '...'. I will investigate it.

File: src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java [hunk 2]
```diff
@@ -341,7 +335,7 @@ public void onFailure( @NotNull Call call, @NotNull IOException e )
                         scope.close();
                         span.end();
 
-                        logger.trace( "Failed: " + call.request().url(), e );
+                        logger.info( "Failed: [" + method + "] " + url, e );
```
Comment (ligangty) [inline src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java:338]: Should this be a warn?

File: src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java [hunk 3]
```diff
@@ -357,7 +351,7 @@ public void onResponse( @NotNull Call call, @NotNull Response response )
                         scope.close();
                         span.end();
 
-                        logger.trace( "Success: " + call.request().url() + " -> " + response.code() );
+                        logger.info( "Success: [" + method + "] " + url + " -> " + response.code() );
```
Comment (ligangty) [inline src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java:354]: I think we should avoid to use "+" for string in logging, instead use parameterized log( use {} ).

File: src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java (hunk 1 above)
Comment (ruhan1) [reply src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java]: I found where it is from. Updated as old indy.

File: src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java (hunk 2 above)
Comment (ruhan1) [reply src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java]: Cool. Updated.

File: src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java (hunk 3 above)
Comment (ruhan1) [reply src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java]: Updated.

File: src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java (hunk 1 above)
Comment (sswguo) [reply src/main/java/org/commonjava/util/gateway/util/WebClientAdapter.java]: That's cool.

**Comment by sswguo** [comment]
LGTM.

**Comment by ruhan1** [comment]
Thanks all for review it.

**Pull request merged**